client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/testdomain/protobuf/marshallers/AccountMarshaller.java

```diff
@@ -35,7 +35,7 @@ public AccountPB readFrom(ProtoStreamReader reader) throws IOException {
       Limits limits = reader.readObject("limits", LimitsPB.class);
       Limits hardLimits = reader.readObject("hardLimits", LimitsPB.class);
       List<byte[]> blurb = reader.readCollection("blurb", new ArrayList<>(), byte[].class);
-      Account.Currency[] currencies = reader.readArray("currencies", Account.Currency.class);
+      List<Account.Currency> currencies = reader.readCollection("currencies", new ArrayList<>(), Account.Currency.class);
```

Comment:
I've seen this in a few places (moving from array to list), what was the issue?

Reply:
Good question. Thanks. I'm checking now why we had to change the modelling of the `getCurrencies` properties...

Reply:
Reverting the changes I didn't find any issue on running tests. So I'm going to restore the original model (array) for such cases.